docs/multi-stage-query/reference.md

```diff
@@ -280,6 +481,31 @@ The following ISO 8601 periods are supported for `TIME_FLOOR`:
 - P3M
 - P1Y
 
+The string constant can also include any of the keywords mentioned above:
+
+- `HOUR` - Same as `'PT1H'`
+- `DAY` - Same as `'P1D'`
```

Comment:
is `WEEK` an option here too?

Reply:
Please don't make WEEK an option.  Kill WEEK, it's not commutable with any other granularity larger than it (MONTH, YEAR, etc.) and generates oddities when used.